### DISCUSSION TITLE
How to run a YT job using a Docker image?

### DISCUSSION THREAD

**Discussion by ibushmarinov** [ytsaurus/ytsaurus#313] [General]
The Russian-language documentation of YTsaurus strongly suggests that there is a way to launch YT jobs using custom filesystem images specified as Docker images:
https://ytsaurus.tech/docs/ru/user-guide/data-processing/porto/layer-paths

Is there a step-by-step guide how to do that?

**Comment by sergey-v-galtsev** [comment]
Unfortunately, this doc is the best we have at the moment.
Also keep in mind that the internal docker registry is not yet included in YTsaurus distribution.

**Comment by koct9i** [comment]
Here is example how cri job environment could be configured right now:
https://github.com/ytsaurus/yt-k8s-operator/pull/86

Better support in yt k8s operator is under development.
